README.md

```diff
@@ -1,59 +1,57 @@
-This project was bootstrapped with
-[Create React App](https://github.com/facebook/create-react-app).
+# Bench Routes Dashboard
 
-## Available Scripts
+Bench Routes Dashboard is a web UI that enables the user to access, visualize, and chart the stored data. This files explains how to setup and work with Bench Routes Dashboard.
 
-In the project directory, you can run:
+## Introduction
 
-### `yarn start`
+The [React-based](https://reactjs.org/) Bench Routes Dashboard was bootstrapped using [Create React App](https://github.com/facebook/create-react-app), a popular toolkit for generating React application setups. You can find general information about Create React App on [their documentation site](https://create-react-app.dev/).
 
-Runs the app in the development mode.<br /> Open
-[http://localhost:3000](http://localhost:3000) to view it in the browser.
+Instead of plain JavaScript, we use [TypeScript](https://www.typescriptlang.org/) to ensure typed code.
 
-The page will reload if you make edits.<br /> You will also see any lint errors
-in the console.
+## Development environment
 
-### `yarn test`
+To work with the React UI code, you will need to have the following tools installed:
 
-Launches the test runner in the interactive watch mode.<br /> See the section
-about
-[running tests](https://facebook.github.io/create-react-app/docs/running-tests)
-for more information.
+- The [Node.js](https://nodejs.org/) JavaScript runtime.
+- The [Yarn](https://yarnpkg.com/) package manager.
+- _Recommended:_ An editor with TypeScript, React, and [ESLint](https://eslint.org/) linting support. See e.g. [Create React App's editor setup instructions](https://create-react-app.dev/docs/setting-up-your-editor/). If you are not sure which editor to use, we recommend using [Visual Studio Code](https://code.visualstudio.com/docs/languages/typescript). Make sure that [the editor uses the project's TypeScript version rather than its own](https://code.visualstudio.com/docs/typescript/typescript-compiling#_using-the-workspace-version-of-typescript).
```

Comment:
Replace with 

_Recommended:_ An editor with TypeScript, React, and [ESLint](https://eslint.org/) linting support(such as vscode). See for e.g. [Create React App's editor setup instructions](https://create-react-app.dev/docs/setting-up-your-editor/). Please ensure that [the editor uses the project's TypeScript version rather than its own](https://code.visualstudio.com/docs/typescript/typescript-compiling#_using-the-workspace-version-of-typescript).